Modules/Utils.cs

```diff
@@ -535,29 +541,32 @@ public static void NotifyRoles(bool isMeeting = false, PlayerControl SpecifySeer
                     //銃声が聞こえるかチェック
                     SelfMark += Sniper.GetShotNotify(seer.PlayerId);
                 }
+
                 //Markとは違い、改行してから追記されます。
                 string SelfSuffix = "";
 
-                if (seer.Is(CustomRoles.BountyHunter) && seer.GetBountyTarget() != null)
+                //他人用の変数定義
+                bool SeerKnowsImpostors = false; //trueの時、インポスターの名前が赤色に見える
+
+                if (seer.Is(CustomRoles.BountyHunter))
```

Comment:
Please change to switch.

example)

switch (seer.GetCustomRole())
{
　case CustomRoles.BountyHunter:
　　　　break;
}

Reply:
Will do